tests/virt_autotest/prepare_alp_host.pm

```diff
@@ -41,6 +44,8 @@ sub test_flags {
 sub post_fail_hook {
     # Let it empty now.
     # Need to let parent one called after yast team modifies the parent class to fit alp.
+    # Upload expected NetworkManager debug log
+    virt_autotest::utils::upload_nm_debug_log;
```

Comment:
@guoxuguang  Hi Leon, during our offline talk this Tuesday, you mentioned that you were considering how to upload failure logs when it fails to set up bridge for whatever reasons, eg network manager bugs. I talked with Julie to see if there can be any experience to be referred from sriov test which has chance for similar network outage during VF operations. The result is that there is NO such handling either in sriov. And we discussed the possibility to use sol console for upload -- possible but no guaranteed way because sol console does not have return code given to openqa. We two agree that if the chance for such issue is low in your validations, we can have this PR merged WITHOUT it. If in official openqa test, it turns out that it happens with possibility higher than acceptable level, you can add failure debug log in post_fail_hook via sol console then. HDYT?

Reply:
Totally agree on this as your mentioned.  thanks so much for your great help and suggestion! I do appreciate that.